corgidrp/data.py

```diff
@@ -794,18 +1044,23 @@ def __init__(self, data_or_filepath, date_valid=None):
         self.params = {}
         # load back in all the values from the header
         for key in self.default_values:
-            self.params[key] = self.ext_hdr[key]
-        
+            if len(key) > 8:
```

Comment:
Not a hard requirements, but might make things simpler: can we just keep the keys to 8 characters? I find headers really hard to read when they include hierarchical keys. Worth a discussion

Reply:
(just to emphasize that this is pretty low priority and is probably fine as is)